gems/pending/appliance_console/database_configuration.rb

```diff
@@ -64,6 +64,9 @@ def activate
 
     def post_activation
       ServiceGroup.new.restart_services
+      evm_service = LinuxAdmin::Service.new("evmserverd")
+      evm_service.enable
+      evm_service.start
```

Comment:
Looks like we should be able to do that
(https://github.com/ManageIQ/linux_admin/blob/master/lib/linux_admin/service/systemd_service.rb#L11)